lib/workers/pr/index.spec.ts

```diff
@@ -4,7 +4,7 @@ import { PLATFORM_TYPE_GITLAB } from '../../constants/platforms';
 import { Pr, platform as _platform } from '../../platform';
 import { BranchStatus } from '../../types';
 import * as _limits from '../global/limits';
-import { BranchConfig, PrResult } from '../types';
+import { BranchConfig } from '../types';
```

Comment:
```suggestion
import type { BranchConfig } from '../types';
```

So those can be type imports now.